import-dashboards.py

```diff
@@ -529,6 +530,20 @@ def set_home_dashboard(api_key):
     #             "SELECT 1, 1, 0, 0, id, '', '', datetime('now'), datetime('now') from dashboard WHERE slug='home'")
 
 
+def remove_dbaas_dashboard(api_key):
+    if DBAAS:
+        print ' * DBaaS is enabled'
+        return
+
+    print ' * DBaaS is disabled'
+    r = requests.get('%s/api/dashboards/uid/pmm-dbaas' % (HOST,), headers=grafana_headers(api_key))
+    if r.status_code == httplib.OK:
+        print '   * Removing DBaaS dashboard'
+        r = requests.delete('%s/api/dashboards/uid/pmm-dbaas' % (HOST,), headers=grafana_headers(api_key))
+        if r.status_code != 200:
```

Comment:
You can use `httplib.OK` here as well 
```suggestion
        if r.status_code != httplib.OK:
```